packages/react-components/react-textarea/src/components/Textarea/useTextareaStyles.styles.ts

```diff
@@ -188,30 +181,30 @@ const useTextareaStyles = makeStyles({
   // The padding style adds both content and regular padding (from design spec), this is because the handle is not
   // affected by changing the padding of the root.
   small: {
-    height: textareaHeight.small,
     minHeight: '40px',
     ...shorthands.padding(
       tokens.spacingVerticalXS,
       `calc(${tokens.spacingHorizontalSNudge} + ${tokens.spacingHorizontalXXS})`,
     ),
+    maxHeight: '200px',
```

Comment:
Are these values specified by design? Where are they coming from?

Reply:
Yes, they come from the figma spec